test/fixtures/instance_template/alias_ip_range/main.tf

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module "instance_template_alias_ip_range" {
+  source          = "../../../../examples/instance_template/alias_ip_range"
+  project_id      = var.project_id
+  subnetwork      = google_compute_subnetwork.main.name
+  service_account = var.service_account
+  tags            = ["foo", "bar"]
+
+  labels = {
+    environment = "dev"
+  }
```

Comment:
Our newer approach is to hard code these in examples as much as possible. In this case we can also drop these since I don't think they provide additional value to the alias ip range example.

Reply:
changed